README.md

```diff
@@ -226,6 +245,7 @@ module.exports = {
 ```
 
 ### Example: Use your own globs
+It's better to use function-based configuration if your use case is this.
```

Comment:
I'd update the example instead of this paragraph

Reply:
What do you mean? Example of function based configuration is already in the documentation. Maybe, a link to that example can be added.